### PR TITLE
Add Duplicati icon

### LINKED ISSUE #10836: Request: Duplicati
### Brand Name

Duplicati

### Website

https://duplicati.com

### Popularity Metric

Global Rank: 443715

### Official Resources for Icon and Color

https://github.com/duplicati/duplicati/blob/2dcafc707b4ae0e936969a5e1ece15043ad4a082/Assets/application%20icon/application_icon.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by anhthang**
image

**Issue:** closes #10836 

**Popularity metric:**

10k ⭐ on Github
https://github.com/duplicati/duplicati

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

SVG extracted from the website, hex value `#1E3A8A` from the main part of the logo

### CODE REVIEW

File: icons/duplicati.svg
Comment (adamrusted) [inline icons/duplicati.svg:1]: Icon shape is a perfect match, but I'm seeing a lot of unnecessary paths left in your version (see below):
image

Would you be able to look at removing those please, @anhthang?

Comment (anhthang) [reply icons/duplicati.svg:1]: @adamrusted just updated, pls review again

Comment (anhthang) [reply icons/duplicati.svg:1]: it seems still have small issue, let me check again

**Review by adamrusted — COMMENTED** [review]
Icon looks to be a perfect match now - though [this page](https://duplicati.com/termsofservice) makes me think we can't include it...

> Trademarks. Company’s name and all related stylizations, graphics, logos, service marks and trade names used on or in connection with any Services are the trademarks of Company and may not be used without permission in connection with your, or any third-party, products or services.

> Certain Restrictions. The rights granted to you in the Agreement are subject to the following restrictions: (b) you shall not frame or utilize framing techniques to enclose any trademark, logo, or other parts of the Services (including images, text, page layout or form);

Thoughts, @uncenter?

**Comment by uncenter** [comment]
I'd have to agree unfortunately. Wording is a bit strange but definitely seems like we can't.

**Comment by adamrusted** [comment]
Before we close this out, did you want to reach out to them, @anhthang? See if we can get permission?

**Comment by anhthang** [comment]
I have sent email to them to request for permission

**Comment by anhthang** [comment]
@adamrusted I think we got green light to go

Screenshot 2024-06-08 at 11 46 59 PM

**Review by adamrusted — COMMENTED** [review]
Perfect match, thanks @anhthang!

**Pull request merged**